src/example.jsx

```diff
@@ -577,6 +578,15 @@ class ExampleApp extends React.PureComponent<Props, State> {
             )}
           </div>
         </article>
+        <article className="gc-panel gc-panel--separator">
+          <header className="gc-panel__title">Dropdown Menu</header>
+          <div className="gc-panel__content">
+            <Dropdown dropdown="dropdown button" items = {[
+            {label:"Color", value:"Red"},
```

Comment:
This `label` is for items, not for the Dropdown label - for dropdown label you should add a separate property for this component.